core/regexp/source_spec.rb

```diff
@@ -7,13 +7,17 @@
     /x(.)xz/.source.should == "x(.)xz"
   end
 
-  it "has US-ASCII encoding when created from an ASCII-only \\u{} literal" do
-    re = /[\u{20}-\u{7E}]/
-    re.source.encoding.should equal(Encoding::US_ASCII)
+  not_compliant_on :opal do
```

Comment:
I am trying to remove `non_compliant_on` guards in general, so I'll switch those to `not_supported_on` (which is the intended meaning here I believe).